dockstore-webservice/src/main/resources/openapi-templates/JodaDateTimeProvider.mustache

```diff
@@ -0,0 +1,44 @@
+package {{apiPackage}};
+
+import com.sun.jersey.core.spi.component.ComponentContext;
+import com.sun.jersey.spi.inject.Injectable;
+import com.sun.jersey.spi.inject.PerRequestTypeInjectableProvider;
+
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.core.UriInfo;
+import javax.ws.rs.ext.Provider;
+import org.joda.time.DateTime;
+import java.util.List;
+
+@Provider
+public class JodaDateTimeProvider extends PerRequestTypeInjectableProvider<QueryParam, DateTime> {
```

Comment:
Without understanding the whole flow and realizing it may be beyond the scope of this PR, do we still need to use Joda? According to [their own website](https://www.joda.org/joda-time/index.html), you only need to be using pre-Java 8.

Reply:
This is part of https://github.com/swagger-api/swagger-codegen/issues/9893  (i.e. all templates need to be included, not just the ones we're using/overriding)

I don't think we've selected Joda time generation, but I could be wrong.